opencti-platform/opencti-graphql/src/modules/support/support.graphql

```diff
@@ -0,0 +1,69 @@
+type SupportPackage implements InternalObject & BasicObject {
+    id: ID!
+    name: String!
+    standard_id: String!
+    entity_type: String!
+    parent_types: [String]!
+    created_at: DateTime!
+    package_status: PackageStatus!
+    package_url: String
+    package_upload_dir: String
+    nodes_status: [SupportNodeStatus]
+    nodes_count: Int!
+    createdBy: Individual
+    creators: [Creator!]
+}
+
+type SupportPackageConnection {
+    pageInfo: PageInfo!
+    edges: [SupportPackageEdge!]!
+}
+
+type SupportNodeStatus {
+    node_id: String!
+    package_status: PackageStatus!
+}
+
+type SupportPackageEdge {
+    cursor: String!
+    node: SupportPackage!
+}
+
+enum PackageStatus {
+    IN_PROGRESS
+    READY
+    IN_ERROR
+}
+
+enum SupportPackageOrdering {
+    name
+    created_at
+    created
+    package_status
+}
+
+input SupportPackageAddInput {
+    name: String! @constraint(minLength: 2)
```

Comment:
Now we also add a "not-blank" constraint for name input (example in administrativeArea.graphql)

Reply:
Could probably also be used for the ID of ForceZip

Reply:
thanks ! I was not aware of this one :)